gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-details/api-tab-details.component.html

```diff
@@ -34,19 +34,16 @@
 
   <div>
     <mat-card appearance="raised" class="api-details__tab-details-info__card">
-      @if (api.version) {
-        <div>
-          <div class="m3-body-large" i18n="@@apiDetailsVersion">Version</div>
-          <div class="m3-body-medium">{{ api.version }}</div>
-        </div>
-      }
-
-      @if (api.updated_at) {
-        <div>
-          <div i18n="@@apiDetailsLastUpdated" class="m3-body-large">API Last Updated</div>
-          <div class="m3-body-medium">{{ api.updated_at | date }}</div>
-        </div>
-      }
+      <div>
+        <ul class="info__miscellaneous">
+          @for (apiInformation of this.apiInformation; track apiInformation.name) {
+            <div>
+              <div class="m3-body-large" i18n="@@apiDetailsVersion">{{ apiInformation.name }}</div>
```

Comment:
Translation not available for dynamic values

```suggestion
              <div class="m3-body-large">{{ apiInformation.name }}</div>
```

Reply:
Done